internal/pluginutil/sdk/api.go

```diff
@@ -47,6 +47,9 @@ func RunCreateAPI(cfg *config.Config) error {
 	return nil
 }
 
+// todo(camilamacedo86): Now that we have the Kubebuilder scaffolding machinery included in our repo, we could make
+// this an actual template that supports both file.Template and file.Inserter for init and create api, respectively.
+// More info: https://github.com/operator-framework/operator-sdk/issues/3370
```

Comment:
We shouldn't be creating issues for internal stuff like this
```suggestion
```

Reply:
See that is a @joelanford recommendation: https://github.com/operator-framework/operator-sdk/pull/3295#discussion_r451274122

Also, I agree with that since give more visibility about what should be done for 1.0.0 and allow us link the issue which makes easier to know what should be done with the todo.

Reply:
The comment you linked seems unrelated to the issue you created.

Reply:
the comment is to ilustrate that IMO we can `be creating issues for internal stuff like this`. however, i think it is sorted out.